metric/system/process/process.go

```diff
@@ -189,6 +211,7 @@ func (procStats *Stats) pidFill(pid int, filter bool) (ProcState, bool, error) {
 	// Fetch proc state so we can get the name for filtering based on user's filter.
 
 	// OS-specific entrypoint, get basic info so we can at least run matchProcess
+	// last non-os-specific call. Here "the problem begins"
```

Comment:
Could this be clarified? It's not clear what problem this means, especially for someone just looking at the code and not this specific PR

Reply:
another debug comment I forgot to delete :man_facepalming: